### PR TITLE
use camelCase variable names

### PR DESCRIPTION
**Pull request by nasermirzaei89**
use camelCase variable names instead of snake_case

### CODE REVIEW

File: query_test.go [hunk 1]
```diff
@@ -54,10 +54,10 @@ type testRow struct {
...
 
 type testRowPtr struct {
-	A_P *int64   `db:"a_p,omitempty"`
-	B_P *string  `db:"b_p,omitempty"`
-	C_P *string  `db:"c_p,omitempty"`
-	D_P *float64 `db:"d_p,omitempty"`
+	AP *int64   `db:"a_p,omitempty"`
+	BP *string  `db:"b_p,omitempty"`
+	CP *string  `db:"c_p,omitempty"`
+	DP *float64 `db:"d_p,omitempty"`
```
Comment (martinrode) [inline query_test.go]: Please leave the "_", this shows clearly that it is a pointer.

may be "pA", "pB" ?

**Review by martinrode — CHANGES_REQUESTED** [review]
I like the cleanup, pls check my comment.

File: query_test.go (hunk 1 above)
Comment (nasermirzaei89) [reply query_test.go]: Really, It doesn't mind. If you think it's better, I can update the commit.

Comment (nasermirzaei89) [reply query_test.go]: You can't Use unexported fields with struct tags. But I can rename them to PointerA to be readable enough.

Comment (martinrode) [reply query_test.go]: Lets do "Ap", "Bp" then.

**Pull request closed without merging**